dev/bin/knative/resources/knative-kafka.yaml

```diff
@@ -0,0 +1,1807 @@
+---
```

Comment:
I'm sure you'll probably tell us all when this is completed, but being impatient where is the `knative-eventing`/`knative-kafka` configured to forward events to our `events-in` [topic](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/executor/src/main/java/com/redhat/service/smartevents/executor/ExecutorsService.java#L30)?

Reply:
@manstis the title of this PR suggests that it's limited to adjusting the pipelines to install knative. The setup of the knative broker is done by the operator here https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/pull/586